yarn-project/end-to-end/scripts/native-network/deploy-l1-contracts.sh

```diff
@@ -27,16 +27,29 @@ export PRIVATE_KEY=${PRIVATE_KEY:-""}
 export SALT=${SALT:-"1337"}
 
 echo "Waiting for Ethereum node to be up..."
-until curl -s -X POST -H 'Content-Type: application/json' \
-  --data '{"jsonrpc":"2.0","method":"eth_blockNumber","params":[],"id":1}' \
-  $ETHEREUM_HOSTS 2>/dev/null | grep -q 'result'; do
+found_node=0
+for HOST in $(echo "${ETHEREUM_HOSTS}" | tr ',' '\n'); do
+  if curl -s -X POST -H 'Content-Type: application/json' \
+    --data '{"jsonrpc":"2.0","method":"eth_blockNumber","params":[],"id":1}' \
+    "$HOST" 2>/dev/null | grep -q 'result'; then
+    echo "Node $HOST is ready"
+    found_node=1
+    break
+  fi
   sleep 1
 done
```

Comment:
Heads up this is not equivalent to the previous one: the new one will exit if the node is not immediately available, while the previous one will retry. Given this is called simultaneously to `ETHEREUM_SCRIPT` in `scripts/run_native_testnet.sh`, we should keep the behaviour of waiting for the node to be up.